server/src/main/java/org/elasticsearch/index/search/stats/SearchStats.java

```diff
@@ -117,6 +117,22 @@ public void add(Stats stats) {
             suggestCurrent += stats.suggestCurrent;
         }
 
+        public void addForClosedShard(Stats stats) {
```

Comment:
perhaps call this `addForClosingShard`. The shard is not closed yet at this point.

Reply:
Changed